README.md

```diff
@@ -28,7 +28,11 @@ Install
 -------
 
 ```
-npm install electron-pdf -g
+`sudo npm install electron-pdf -g`
+
+Note: If you're installing electron-pdf using the system level npm (vs a user-level install like with NVM) then you may need to run the following command instead:
+`sudo npm install electron-pdf -g --unsafe-perm`
```

Comment:
This isn't going to be rendered correctly with markdown. It should be like just like this:
My apologies for making this confusing in my earlier comment.

```
npm install electron-pdf
```

Note: If you're installing electron-pdf as root using the system level npm (vs a user-level install like with NVM) then you may need to run the following command instead:
```
sudo npm install electron-pdf -g --unsafe-perm
```
Please see [the npm docs](https://docs.npmjs.com/misc/config#unsafe-perm) for more information.